components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add-finish-ajaxprocessor.jsp

```diff
@@ -98,11 +100,20 @@
                     buff.append(grantType + " ");
                 }
             }
-
             grants = buff.toString();
 
+            List<String> registeredScopeValidators = new ArrayList<String>();
+            String[] allowedValidators = client.getAllowedScopeValidators();
+            for (String allowedValidator : allowedValidators) {
+                String val = request.getParameter("scope_validator_" + allowedValidator);
```

Comment:
Can't we use constant for "scope_validator_" since it has been used in more places.